sonar-scanner/bin/preview

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+echo This is a wrapper for \`$ sonar-scanner\` with default arguments to run in GitLab CI
+echo It takes one argument: a comma-separated of directories to check, such as: \`$ scan kw,cron,scripts\`
+echo If you want to customize it, run \`$ sonar-scanner \<your-arguments\>\` instead.
```

Comment:
Wouldn't
> If you want to customize it, run `sonar-scanner kw,cron,script \<additional-arguments\>\` 

be a bit more specific in this case? Is it really intended to use as "instead"? |
Could you plz provide some use case?

Reply:
No, check the script. `preview` is calling `sonar-scanner` with a bunch of preset arguments. `sonar-scanner kw,cron,script` will not work.

The usecase here is to be able to use this image if the user does not use GitLab CI or our standards.